doc/chapters/gateways/10-dbus-gateway.rst

```diff
@@ -19,10 +19,25 @@ communication, and the application running within the container is
 expected to use these variables (probably by means of the binding used).
```

Comment:
Could you please take the opportunity to remove the extra word in the above sentence (second paragraph, 3rd line from the bottom): "The gateway uses **uses** an"